doc/ale-rust.txt

```diff
@@ -59,6 +59,35 @@ g:ale_rust_cargo_check_all_targets         *g:ale_rust_cargo_check_all_targets*
   is used. See |g:ale_rust_cargo_use_check|,
 
 
+g:ale_rust_cargo_default_feature_behavior
+                                    *g:ale_rust_cargo_default_feature_behavior*
+                                    *b:ale_rust_cargo_default_feature_behavior*
+  Type: |Number|
+  Default: `1`
+
+  When set to `0`, ALE will set the `--no-default-features` option when
+  invoking `cargo`. Only the features specified in
+  |g:ale_rust_cargo_include_features| will be included when performing the
+  lint check.
+
+  When set to `1`, ALE will instruct `cargo` to build all default features
```

Comment:
Use strings instead of numbers like `1` or `2`. Otherwise you need a comment in your vimrc file to explain what `2` means.

Reply:
Updated the `0`, `1`, `2` options to `none`, `default`, and `all` respectively.